pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -506,7 +506,7 @@ private void LoadLayoutLocked(Stream layoutStream)
             _listGraphDetections.ToList().ForEach(DestroyGraphDetections);
             DetectionsGraphController.GraphType = type;
 
-            FormUtil.OpenForms.OfType<FoldChangeForm>().ForEach(f => f.Close());
+            CollectionUtil.ForEach(FormUtil.OpenForms.OfType<FoldChangeForm>(), f => f.Close());
```

Comment:
This is a strange change, I don't remember doing it. Might has come from a master merge.